### ISSUE TITLE
The EOH overclocking algorithm has serious problems, resulting in high overclocking benefits and extremely low costs

### ISSUE THREAD

**Issue by JodieRuth** [GTNewHorizons/GT-New-Horizons-Modpack#13788]
### Your GTNH Discord Username

JodieRuth

### Your Pack Version

2.3.4

### Your Proposal

After modifying the existing overclocking algorithm of EOH, overclocking will not continue to increase after reaching a certain threshold, but will in turn reduce power consumption. Under 24 overclocking, the single working time of Level 1 EOH can be reduced to 1 tick at an extremely low cost, or the working time of Level 10 EOH can be reduced to just a few seconds without paying any cost, just some electricity, resulting in a significant reduction in waiting time and material output thousands of times higher than before
EOH现有的超频算法修改后，超频到一定阈值之后不会继续增加而是反过来减少耗电，在24超频下能以极低的代价将1级EOH的单次工作时间变为1tick或者将10级EOH的工作时间变为区区几秒，而不需要付出什么代价仅仅只是一些电量，导致了等待时间大幅减少，材料产出变为以前的数千倍

### Your Goal

If this overfrequency algorithm is not modified, the future EOH will change from a generator to a mining and resource production machine with extremely powerful output and speed, producing a large number of Almost all minerals in the game and producing a large number of exclusive materials for the UXV stage, which is obviously unreasonable
如果不修改这个超频算法，以后的EOH将会从一个发电机变成一个产量和速度都极为强大的采矿与资源生产机器，大量生产游戏内几乎所有矿物的同时出产大量UXV阶段专属材料，这很明显是不合理的

### Your Vision

Revise the EOH overclocking algorithm to give EOH a reasonable overclocking instead of two unreasonable algorithms like the first or current version
重新修改EOH超频算法，给予EOH一个合理的超频而不是与第一版或者当前版本这样两个不合理的算法

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by Gofer2x** [comment]
Yeah, it's certainly a weird design choice but apparently intentional? Check this issue I made. https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/13770

**Comment by JodieRuth** [comment]
No, I don't think they would make such a low-level mistake. Using an EOH with 24 overclocking cycles can complete the work within 1 tick, and even a 10 level EOH has almost no operational cost, consuming EU that is completely different from the product, which is clearly a bug

**Comment by JodieRuth** [comment]
After that, you only need about 3 to 5 days to upgrade from level 1 EOH to level 10 EOH

**Comment by TheEpicGamer274** [comment]
it would be more than 5 days I bet, because the processing times for the raw amount of stuff required for each tier would take atleast 1-2 weeks of crafting to do it I bet

**Comment by JodieRuth** [comment]
No, the production time of other raw materials is completely insignificant compared to EOH exclusive materials. In most cases, the speed of upgrading EOH will be blocked by the production speed of EOH exclusive materials. This overclocking formula has changed this and turned the full level EOH into a terrifying monster

**Comment by JodieRuth** [comment]
When using 24 overclocking cycles, you only pay a few thousandths of the original formula's required EU, or even lower

**Comment by TheEpicGamer274** [comment]
how about instead of saying all this stuff, give a proper eu/t amount of all the ocs, no point of discussing anything if I can't see if the eu/t amount is proper or not.

**Comment by JodieRuth** [comment]
Level 1 EOH, placed on Earth, will reach a speed of 1 tick during 22 overclocking cycles, using approximately 5e15EU, while 24 overclocking cycles have the same speed, using approximately 2.4e15EU

**Comment by TheEpicGamer274** [comment]
After discussion with some other endgame players with EOH, it does seem that the formula does need some reworking, hopefully it should be fixed soon or might never get fixed, I don't really have a clue.

**Comment by JodieRuth** [comment]
Another issue with the overclocking formula is that it brings a very fast material production speed. I think Colen will definitely change it, just like changing the production time of the EOH controller. Under this new formula, I can produce almost all the minerals and materials in the UXV stage in the game at a very fast speed, far more than all other machines. I can produce 1e10 or even more spacetime and other useful things in one night. Compared with its production capacity, the EU used seems to be no longer important

**Comment by TheEpicGamer274** [comment]
Yeah I've dm'ed the guy who made the buffed formula, hopefully we can do some revisions tmrw.
